plugins/woocommerce-admin/client/products/product-page.scss

```diff
@@ -61,10 +62,20 @@
 	}
 }
 
+.woocommerce-admin-page__product_product-id_variation_variation-id {
+	.interface-interface-skeleton__content {
+		overflow: hidden;
```

Comment:
For some odd reason this was necessary on the product variation edit screen to prevent a double scroll bar from showing. To my knowledge all the other CSS should be identical to the main product edit screen, so I have no clue what changed that caused the double scrollbar to show up here again and not in the edit product screen.
This worked, although let me know if you find the real reason (if there is one?)